### DISCUSSION TITLE
can't upload picture

### DISCUSSION THREAD

**Discussion by hansemso** [community/community#43848] [New to GitHub] [answered]
Hi. I'm building my website and I can't upload a picture of my cat.  At line 64 I put 
       <p class="w3-border w3-padding-large w3-padding-64 w3-center"><img src="https://drive.google.com/file/d/1-j3emMzfk4OLgK8oyTqgWq2jK1p9F0ZH/view?usp=share_link"></p>
but all I see is the tiny no photo icon.  What am I going wrong?

**Comment by ihnatenkoo** [comment]
Your photo on google disk is сlosed for viewing, you should open access

**Comment by radar-blip** [comment]
You should be able to upload the image to your repository and make a link to it.

![upload](https://user-images.githubusercontent.com/39196767/211636655-a5fe4c2f-7ca3-46aa-a0a4-222d2674f9eb.png)

**Comment by hansemso** [comment]
Ah, ok.  I will look into it.  Thanks!

**Reply by hansemso** [reply]
Well, I uploaded one.  But it won't fit.  How do I resize it?

**Reply by ihnatenkoo** [reply]
I maked some changes, So fit?
![3123123](https://user-images.githubusercontent.com/89586345/211842989-2368d99f-5889-4de4-9830-9887cb8c3b30.png)

**Reply by hansemso** [reply]
Yes, how do I do that?

**Reply by radar-blip** [reply]
You need to use an image editing tool like photoshop (or similar) to scale the size of image down.

**Comment by hansemso** [comment]
I see.  I'll look into it.  Thanks.  Is there a way I can change the size of the windows?  There are 4 for 4 ads.  I want to delete 2 and enlarge the other two.

**Comment by ihnatenkoo** [comment]
Add style="max-width:100%" in your img tag
`<img style="max-width:100%" src='me_zoomed_out.jpg' alt='photo'/>`

**Reply by hansemso** [reply]
I did that, and it still doesn't work.  Still has the little no photo icon.  What am I doing wrong?

**Reply by ihnatenkoo** [reply]
Wrong src, path to the photo.
me_zoomed_out.jpg doesn't exist in your repository, add to repository or load photo to the internet and use URL path in src attribute

**Comment by hansemso** [comment, marked as answer]
Well, it's in the webpage but again, it won't scale properly.  The style-max-width=100% command isn't working.

**Reply by ihnatenkoo** [reply]
`style="max-width:100%"`

**Reply by hansemso** [reply]
The link in the email didn't work.  I tried copy-paste, didn't work.  Not sure how to do a pull request, I'll have to look for a tutorial.  I tried it with ...max-width: auto" too but still not working.

**Reply by hansemso** [reply]
dunno . . . now it all of a sudden works.  Thanks, you guys are great  :)

**Reply by hansemso** [reply]
i did already and don't trust ukrainian seinfeld

**Reply by ihnatenkoo** [reply]
ok I don't help you anymore

**Comment by hansemso** [comment]
i understand.  peace  :)
